src/main/groovy/au/org/ala/scatterplot/Scatterplot.java

```diff
@@ -1131,7 +1136,7 @@ public Paint getPaint(double value) {
 
         String[] layers = new String[layersToSample.size()];
         layersToSample.toArray(layers);
-
+        logger.info("Sampling points... ");
```

Comment:
also use log() to append this message to the task history.

Reply:
It is a independent JAVA class, are not inherited from SlaveProcess. I cannot find taskLog()